azure/services/managedclusters/spec.go

```diff
@@ -113,6 +113,21 @@ type ManagedClusterSpec struct {
 
 	// KubeletUserAssignedIdentity is the user-assigned identity for kubelet to authenticate to ACR.
 	KubeletUserAssignedIdentity string
+
+	// HTTPProxyConfig is the HTTP proxy configuration for the cluster.
+	HTTPProxyConfig *HTTPProxyConfig
+}
+
+// HTTPProxyConfig is the HTTP proxy configuration for the cluster.
+type HTTPProxyConfig struct {
+	// HTTPProxy - The HTTP proxy server endpoint to use.
```

Comment:
```suggestion

	// HTTPProxy is the HTTP proxy server endpoint to use.
```

Could we add newlines between the fields as well as reword it from `SomeField - Something` to `SomeField is something`?